src/js/factfinder/search-redirect.js

```diff
@@ -1,9 +1,12 @@
 document.addEventListener('ffReady', function () {
     var redirectPath = window.ffRedirectPath || '/factfinder/result';
+    var isSearchResultPage = function () {
+        return window.location.href.indexOf(redirectPath) >= 0;
+    };
 
     factfinder.communication.EventAggregator.addBeforeDispatchingCallback(function (event) {
-        if (event.type === 'search' && !event.__immediate) {
-            delete event.type;
+        if (event.type === 'search' && !isSearchResultPage() && !event.searchImmediate && event.navigation !== 'true') {
```

Comment:
I logged to the console, `navigation: ['true']` - why did we put it here again? Was it something with the ASN or the price filters? I can't remember...

Reply:
It was because of clicking on slider type ASN group emits search events, not filter event